javaCarAPI/src/test/java/apitest/MockCar.java

```diff
@@ -0,0 +1,77 @@
+package apitest;
+
+import org.eclipse.paho.client.mqttv3.MqttClient;
+import org.eclipse.paho.client.mqttv3.MqttConnectOptions;
+import org.eclipse.paho.client.mqttv3.MqttException;
+import org.eclipse.paho.client.mqttv3.MqttMessage;
+import org.eclipse.paho.client.mqttv3.persist.MqttDefaultFilePersistence;
+import api.sensor.Infrared;
+import api.sensor.Odometer;
+
+import java.nio.charset.StandardCharsets;
+import java.util.UUID;
+
+public class MockCar {
+
+    private final MqttClient client;
+    private final String carName;
+
+    private String ultrasonicFront;
```

Comment:
Consider renaming the mqtt topic variables to something that better conveys their purpose.